src/Manifest/FileBucketManifestReader.php

```diff
@@ -40,7 +43,14 @@ public function readManifests(int $releaseGroupId): array
             return [];
         }
 
-        return json_decode($fileContent, true);
+        $manifests = json_decode($fileContent, true);
+        if (!is_array($manifests)) {
+            throw new RuntimeException(
+                sprintf('Invalis manifest data, release group ID `%s`, data `%s`', $releaseGroupId, $fileContent),
```

Comment:
typo Invalid
also, maybe we should not list the data, as it could be too long for logs.

Reply:
Thanks ++